src/BlockchainWriter/BlockInfoDAO.test.ts

```diff
@@ -0,0 +1,49 @@
+/* tslint:disable:no-relative-imports */
+import { describe } from 'riteway'
+import { runtimeId, setUpDb } from '../../tests/helpers/utils'
+import { BlockInfoDAO } from './BlockInfoDAO'
+
+const prefix = runtimeId()
```

Comment:
@warrenv is this a unit test or an integration test?

Reply:
So here's my plan:
1. Keep this as a unit test until it's wired up in the router.
2. Once that's done, move this to an integration test
3. Finally, once #707 is done, move it into a functional test.

Reply:
Got it.